addons/sale_project/models/sale_order_line.py

```diff
@@ -188,22 +192,33 @@ def _timesheet_create_project_prepare_values(self):
             'user_id': self.product_id.project_template_id.user_id.id,
         }
 
+    def _set_analytic_distribution(self, inv_line_vals):
```

Comment:
I must admit I have difficulty understanding what this is trying to improve as a flow.
In which case does a SOL not have a distribution while having a project_id on the order?
As far as I can see, that's if the user removed it on purpose. 
But then, why are we re-adding it to the invoice line?

Reply:
Indeed, we tried to mimic the previous behavior where the analytic account of the SO was transferred to the invoice line at invoice creation. But I don't think it's necessary anymore because as you pointed out, the distribution of the project is now given to the SOL so I think we will put back this method in sale module and remove the part where we add analytic_account_id to the distribution of the invoice line